sources/helpers/permissions.js

```diff
@@ -0,0 +1,38 @@
+const Users = require('../models/user.js');
+
+const rolesPermissionsMap={
+    "1":["users.Auth","users.all","users.Update"],
+    "2":["users.secretEvents"],
+    "3":["stories.add"]
+}
+
+const populatePermissions=async (id)=>{
+    const foundUser= await Users.findById(id).populate("clubAccess");
+    let rolesSet=new Set();
+    foundUser.clubAccess.forEach(accessObj => {
+        if(accessObj.club)
+            rolesSet.add(accessObj.level+"$"+accessObj.club)
```

Comment:
The encoding scheme isn't required, we can use an object with key value pairs.

Reply:
i will merge now, encoding algorithm could be changed later